pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>0.1</version>
 
     <properties>
-        <java.version>1.7</java.version>
+        <java.version>1.8</java.version>
```

Comment:
The Travis script needs to be fixed to stop failing when compiling against oraclejdk7 and openjdk7.